.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": false
```

Comment:
Just wondering what's the reason for this setting?

Reply:
Ahrg! My bad. Commited by mistake 😢 Removed by https://github.com/zoltantothcom/Design-Patterns-JavaScript/pull/68/commits/184d978ff064f1610b2b41975ae0050076d89e5d